### DISCUSSION TITLE
'Schema.NET.IHospital.OpeningHours' collides with another property.

### DISCUSSION THREAD

**Discussion by fewless2019** [RehanSaeed/Schema.NET#602] [Q&A]
I have been coming across this error recently version 12.0.0-> 'Schema.NET.IHospital.OpeningHours' collides with another property."

I have found the same two properties in different classes in this solution. 

hospital -> CivicStructureAndEmergencyServiceAndMedicalOrganization -> 
[JsonPropertyName("openingHours")]
[JsonPropertyOrder(307)]
[JsonConverter(typeof(ValuesJsonConverter))]
public override OneOrMany<string> OpeningHours { get; set; }

Physician-> MedicalBusinessAndMedicalOrganization -> LocalBusinessAndOrganization -> 
[JsonPropertyName("openingHours")]
[JsonPropertyOrder(238)]
[JsonConverter(typeof(ValuesJsonConverter))]
public OneOrMany<string> OpeningHours { get; set; }

In my case both of these objects are empty, but this looks to be causing the issue. 
Has any one found a way to fix this?

**Comment by RehanSaeed** [comment]
Are you getting a build error when building the solution? Sorry, not sure I fully understand.

**Comment by RehanSaeed** [comment]
Closing this in favour of https://github.com/RehanSaeed/Schema.NET/issues/603
